pkgs/applications/graphics/feh/default.nix

```diff
@@ -14,6 +14,11 @@ stdenv.mkDerivation rec {
     hash = "sha256-9NJ6zgQHcFJPmRlqJuCMXcKjLvDPUG+QvKGTJlWvWK4=";
   };
 
+  patches = [
+    # upstream PR: https://github.com/derf/feh/pull/723
+    ./0001-Right-click-crash-fix.patch
```

Comment:
Please use `fetchpath` instead (on https://github.com/derf/feh/commit/2c31f8863b80030e772a529ade519fc2fee4a991.patch)

Reply:
Already in the patch I sent

Reply:
I have added your changes.